app/src/main/java/net/skyscanner/backpack/demo/compose/CardStory.kt

```diff
@@ -0,0 +1,73 @@
+package net.skyscanner.backpack.demo.compose
+
+import androidx.compose.foundation.interaction.FocusInteraction
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.collectIsFocusedAsState
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import net.skyscanner.backpack.compose.card.BpkCard
+import net.skyscanner.backpack.compose.card.BpkCardCorner
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+
+@Composable
+fun CardStory() {
+  Column(
+    modifier = Modifier.padding(horizontal = BpkSpacing.Base, vertical = BpkSpacing.Xxl),
+    verticalArrangement = Arrangement.spacedBy(BpkSpacing.Base),
+  ) {
+
+    BpkCard(
```

Comment:
can the individual cards be separate functions with preview to make it easier to read?

Reply:
It will not always be possible. For instance, in this particial case we have card wich switches between focused and non-focused state – if we extract it as a preview, it will display only one state be default.

Reply:
that's fine - still better to have a few separate functions to describe what they are